src/sage/arith/misc.py

```diff
@@ -1970,12 +1970,22 @@ def xgcd(a, b):
 
     INPUT:
 
+    One of the following:
+
     -  ``a, b`` -- integers or more generally, element of a ring for which the
        xgcd make sense (e.g. a field or univariate polynomials).
 
+    -  ``a`` -- list or tuple of at least two integers or more generally, element
+       of a ring which the xgcd make sense.
+
     OUTPUT:
 
-    -  ``g, s, t`` -- such that `g = s\cdot a + t\cdot b`
+    One of the following:
+
+    -  ``g, s, t`` -- when two inputs ``a, b`` are given. They satisfy `g = s\cdot a + t\cdot b`.
+
+    -  ``r`` -- when only ``a`` is given (and ``b = None``). The tuple ``r`` has length one longer
+       than the length of ``a``, and its entries satisfy ``r[0] = \sum_{0 \le i < len(a)} r[i + 1] * a[i]``.
```

Comment:
```suggestion
       than the length of ``a``, and its entries satisfy `r_0 = \sum_{i = 0}^{len(a) - 1} r_{i + 1}a_i`.
```
Single backticks means latex, double backticks means code blocks. Here you probably want latex as a math equation.